commands/info.js

```diff
@@ -56,7 +61,7 @@ function * run (context, heroku) {
         path: `/client/v11/databases/${addon.name}`
       }).catch(err => {
         if (err.statusCode !== 404) throw err
-        cli.warn(`${cli.color.addon(addon.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku pg:wait')} to wait until the db is provisioned.`)
+        cli.warn(`${cli.color.addon(addon.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku addons:wait')} to wait until the db is provisioned.`)
```

Comment:
I also meant to look into more for exactly what situation `pg:wait` doesn't work well and potentially fix it, but I also have some feeling that it may not be an easy fix.